### ISSUE TITLE
Is 'name_de' not a waste of space ?

### ISSUE THREAD

**Issue by asianrider** [openmaptiles/openmaptiles#769]
I was wondering if this tag is relevant. I like the idea of having name_en available as English is the de facto OSM and international language, but, besides the fact that the original creators were german-speaking (kinda ;-), it seems like a waste of space as name:de is already available and coalescing can be made more easily at the SDK level (with mapbox sdk at least).

Danke für ihr Verständnis ;-)

**Comment by nyurik** [comment]
While PBFs makes it very efficient to store repeated identical values (each unique string is stored only once), I was also thinking that we should remove those names that we had for historical reasons just for the sake of clarity and compactness.  I don't even think we should have `name_en` -- we could have `name_int` (for latin-based international name), but all language-specific ones should be `name:<code>`.

**Comment by asianrider** [comment]
OK, I didn't know about this PBF optimization. 
Yes, you're right, name_int is a better candidate for this. I can submit a PR if wanted.

**Comment by klokan** [comment]
This is a breaking change for schema 3.x - if merged and released - people who upgrade need to update their styles.

One of the things we wanted to do on 4.x schema change (in a semver style).

If to do that - then removing all "name_[language]" perhaps, not just "name_de" - as they are replaceable with "name:[language]" and name_int + name_latin.

**Comment by nyurik** [comment]
I agree - we should remove `name_en` at the same time. What's more, I think we have some weird `name:latin` and `name:nonlatin` hack, which I think we should get rid of too -- `name:xx` should only be used by the **real** languages, and everything else like latinizated name should be using the underscore (i.e. `name_latin`).

Currently, the OMT-Tools add `name:latin` and `name:nonlatin` - see [tileset.py](https://github.com/openmaptiles/openmaptiles-tools/blob/a87d3b97dc1bc009d00df877c66b6ca983fc2a98/openmaptiles/tileset.py#L226) and [language.sql](https://github.com/openmaptiles/openmaptiles-tools/blob/5f3c2984608ae3696bfcbe4f8a3c3b19c897ea16/sql/zzz_language.sql#L107).

**Comment by ZeLonewolf** [comment]
Can we add this to the milestone list for 4.0?
